Wikipedia/Code/PreviewLicenseView.m

```diff
@@ -32,7 +32,7 @@ - (void)didMoveToSuperview {
     self.licenseTitleLabel.font = [UIFont systemFontOfSize:11.0];
     self.licenseLoginLabel.font = [UIFont systemFontOfSize:11.0];
 
-    self.licenseTitleLabel.text = WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-terms-and-license", nil, nil, @"By publishing, you agree to the %1$@, and to irrevocably release your contributions under the %2$@ license.", @"Button text for information about the Terms of Use and edit license. Parameters:\n* %1$@ - 'Terms of Use' link ([[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-name]])\n* %2$@ - license name link");
+    self.licenseTitleLabel.text = WMFLocalizedStringWithDefaultValue(@"wikitext-upload-save-terms-and-license", nil, nil, @"By publishing changes, you agree to the %1$@and agree to release your contribution under the %2$@ and %3$@ license.", @"Button text for information about the Terms of Use and edit licenses. Parameters:\n* %1$@ - 'Terms of Use' link ([[Wikimedia:Wikipedia-ios-wikitext-upload-save-terms-name]])\n* %2$@ - license name link 1\n* %3$@ - license name link 2");
```

Comment:
From the mobile web's preview page (with the only difference being publish instead of save)